packages/playwright/src/transform/transform.ts

```diff
@@ -174,7 +183,7 @@ export function transformHook(originalCode: string, filename: string, moduleUrl?
   process.env.BROWSERSLIST_IGNORE_OLD_DATA = 'true';
 
   const { babelTransform }: { babelTransform: BabelTransformFunction } = require('./babelBundle');
-  const { code, map } = babelTransform(originalCode, filename, isTypeScript, !!moduleUrl, pluginsPrologue, pluginsEpilogue);
+  const { code, map } = babelTransform(originalCode, filename, { isTypeScript, isModule: !!moduleUrl, pluginsPrologue, pluginsEpilogue, jsx: tsconfig?.jsx, jsxFactory: tsconfig?.jsxFactory, jsxFragmentFactory: tsconfig?.jsxFragmentFactory, jsxImportSource: tsconfig?.jsxImportSource });
```

Comment:
These now need to be a part of the `calculateHash` below since they affect the compilation result.